include/pipert/UDPConnection.h

```diff
@@ -0,0 +1,47 @@
+#ifndef _UDPCONNECTION_H_
```

Comment:
If you move a file to a different place, use git mv instead of removal and insertion as it looses its history. However, I would be against moving this to the public API.